src/api/query-helpers.ts

```diff
@@ -201,3 +202,23 @@ export function parseTraitAbi(req: Request, res: Response, next: NextFunction):
   }
   handleBadRequest(res, next, `Invalid 'trait_abi'`);
 }
+
+export function validateRequestHexInput(hash: string) {
+  try {
+    const buffer = hexToBuffer(hash);
+    if (buffer.toString() === '') {
+      throw new Error('Invalid hash characters');
```

Comment:
How does this validate the hex string? I left a couple examples of how to validate the string in my previous comment.